### PR TITLE
Arabic updates, fix channels, remove duplicates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: channels/ca.m3u [hunk 1]
```diff
@@ -35,8 +35,8 @@ http://cdn8.live247stream.com/canadaone/tv/playlist.m3u8
 http://live.canadastartv.com:1935/canadastartv/canadastartv/playlist.m3u
 #EXTINF:-1 tvg-id="CanadianArabTV.ca" tvg-name="Canadian Arab TV" tvg-country="CA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/rAZBgir.png" group-title="",Canadian Arab TV
 http://142.112.39.133:8080/catv.mp4
-#EXTINF:-1 tvg-id="CanadianArabTV.ca" tvg-name="Canadian Arab TV" tvg-country="CA" tvg-language="" tvg-logo="" group-title="",Canadian Arab TV
-http://192.226.228.32:8080/hls/catv/index.m3u8
```
Comment (freearhey) [inline channels/ca.m3u:39]: Working link.

File: channels/sa.m3u [hunk 2]
```diff
@@ -1,11 +1,7 @@
 #EXTM3U x-tvg-url="https://raw.githubusercontent.com/Fazzani/grab/master/merge.zip"
-#EXTINF:-1 tvg-id="makkah.sa" tvg-name="-makkah" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",-makkah
-http://mbningestworld-i.akamaihd.net/hls/live/644021/iraqworldsafe/master_2596.m3u8
```
Comment (freearhey) [inline channels/sa.m3u:3]: Working link. Not duplicate.

File: channels/sa.m3u [hunk 3]
```diff
@@ -35,16 +31,10 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/saudi_sunnah/hls1/saudi_
...
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (1080p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_1080p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (240p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_240p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (360p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_360p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (480p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_480p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (720p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_720p/chunks.m3u8
```
Comment (freearhey) [inline channels/sa.m3u:47]: Working links. Not duplicates.

**Review by freearhey — CHANGES_REQUESTED** [review]
First, we consider transactions as duplicates only if they come from the same domain and do not differ in resolution.

Secondly, we do not remove working links from the playlist. If necessary, you can add an alternative broadcast to the end of the file.

Otherwise, it's fine.

File: channels/sa.m3u (hunk 2 above)
Comment (tilfizyawn) [reply channels/sa.m3u:3]: It's Al Hurra Iraq which doesn't belong in sa.m3u. It's already in iq.m3u

https://github.com/iptv-org/iptv/blob/86fbb63ea771d3445220235c42ea842bf33952d2/channels/iq.m3u#L7

File: channels/ca.m3u (hunk 1 above)
Comment (tilfizyawn) [reply channels/ca.m3u]: Reverted

File: channels/sa.m3u (hunk 3 above)
Comment (tilfizyawn) [reply channels/sa.m3u]: They don't work for me but I added them back

**Comment by tilfizyawn** [comment]
@freearhey Thanks for this project and your feedback. I have made the changes, the problem is some mirrors (like Makkah) are the same resolution.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@tilfizyawn Thank you for your help!

**Pull request merged**